### ISSUE TITLE
If graphics are set to Fabulous when shaders are disabled, enabling and then disabling shaders resets graphics option to Fancy

### ISSUE THREAD

**Issue by CalXee** [IrisShaders/Iris#860]
### What happened?

Title says it all. It's just annoying for the minority who play on Fabulous. https://youtu.be/I1IOr_w8iko

### Iris and Minecraft Version

Minecraft 1.17.1, Iris 1.1.2

### Are you running Sodium along with Iris?

Yes

### Operating System

Windows 10

### What is your GPU?

GTX 1050 Ti

### Java Version

Java 16

**Comment by jmaximusix** [comment]
As far as I know the "Fabulous" option isn't compatible with shaders in general (same for optifine). So it might be intended behaviour to reset this option

**Comment by Mr-Waterscout** [comment]
Yeah, Optifine gives you a message saying you can't run shaders with the "Fabulous" graphics option. I'm surprised Iris didn't tell you that along with changing the graphics option to "Fancy".

**Comment by AlsoNiko** [comment]
This is definitely not an intended behavior. No, the shaders aren't running on fabulous, it auto sets to fancy and disables fabulous when shaders are enabled, however enabling shaders SHOULD revert back to fabulous, not fancy.

**Comment by IMS212** [comment]
The issue here is enabling shaders resets your previous option to Fancy from Fabulous, instead of reverting to what you had last.

**Comment by peow** [comment]
im using Iris 1.7.0 on Fabric 1.20.1 and this problem still persists :(
